src/algebra/ordered_ring.lean

```diff
@@ -763,9 +752,14 @@ let s : linear_ordered_semiring α := @linear_ordered_ring.to_linear_ordered_sem
   mul_lt_mul_of_pos_right    := @linear_ordered_semiring.mul_lt_mul_of_pos_right α s,
   ..d }
 
-section decidable_linear_ordered_comm_ring
+instance linear_ordered_comm_ring.to_linear_ordered_add_comm_group
+  [s : linear_ordered_comm_ring α] :
+  linear_ordered_add_comm_group α :=
+{ .. s }
```

Comment:
Is this needed? (would it be inferred automatically?)

Reply:
No, `linear_ordered_comm_ring` doesn't extend `linear_ordered_add_comm_group`.

Reply:
Removed this, added `linear_ordered_ring.to_linear_ordered_add_comm_group` instead.